src/containers/shared/components/Transaction/NFTokenMint/test/NFTokenMintSimple.test.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import { BrowserRouter as Router } from 'react-router-dom'
+import { mount } from 'enzyme'
+import { I18nextProvider } from 'react-i18next'
+import { Simple as NFTokenMint } from '../Simple'
+import transaction1 from './mock_data/NFTokenMint1.json'
```

Comment:
Not a fan of the names of these files.  They dont explain what these example transactions have that make them special.